cmd/maestro/server/api_server.go

```diff
@@ -119,6 +121,16 @@ func NewAPIServer(eventBroadcaster *event.EventBroadcaster) Server {
 	)(mainHandler)
 
 	mainHandler = removeTrailingSlash(mainHandler)
+	mainHandler = traceAttributeMiddleware(mainHandler)
```

Comment:
Like we did for CS, we can add more attributes to the span in a follow-up PR to keep things simple.